### PR TITLE
use shorter uuids for paths

### LINKED ISSUE #121: Waiting for server to be online...
I don’t use [brew](https://brew.sh/) so I installed 3mux using
```
> go get github.com/aaronjanse/3mux
```
and when I try to run 3mux it asks me for a session name (I entered `test`) and I get this:
```
> ./go/bin/3mux
Waiting for server to be online... (/var/folders/d5/6_9wptzx0ml6ys098gqncydm0000gn/T/3mux/92c9afcc-e1f5-4d81-5821-fad5a3ffae1e/fd.sock)
Although the server socket exists, connection to it failed: dial unix /var/folders/d5/6_9wptzx0ml6ys098gqncydm0000gn/T/3mux/92c9afcc-e1f5-4d81-5821-fad5a3ffae1e/fd.sock: connect: connection refused
See server-side logs at /var/folders/d5/6_9wptzx0ml6ys098gqncydm0000gn/T/3mux/92c9afcc-e1f5-4d81-5821-fad5a3ffae1e/logs-server.txt
To manually kill this session, run `3mux kill test`
```
where the log contains:
```
> sudo cat /var/folders/d5/6_9wptzx0ml6ys098gqncydm0000gn/T/3mux/92c9afcc-e1f5-4d81-5821-fad5a3ffae1e/logs-server.txt
2021/03/26 11:17:45 Booting...
2021/03/26 11:17:45 Detach error: listen unix /var/folders/d5/6_9wptzx0ml6ys098gqncydm0000gn/T/3mux/92c9afcc-e1f5-4d81-5821-fad5a3ffae1e/detach-server.sock: bind: invalid argument
```
So, my question is: what server is expected to run here?

### PR DESCRIPTION
**Pull request by aaronjanse**
Fixes #121

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -551,3 +551,19 @@ func surveyName(stdin chan ecma48.Output) string {
 		}
 	}
 }
+
+// log2(36^16) = 82 bits of entropy should be enough
+const randIdentChars = "abcdefghijklmnopqrstuvwxyz0123456789"
+const randIdentLen = 16
```
Comment (PotatoParser) [inline main.go:557]: Hmmm, I think you could go with `base64url` characters (A-Z, a-z, 0-9, -. _), and reduce randIdentLen to 14 to fit in a few less characters :D

**Review by PotatoParser — APPROVED** [review]
Looks good! I think the uuids could be shortened just a bit further :D

File: main.go (hunk 1 above)
Comment (aaronjanse) [reply main.go:557]: Ah, note that some filesystems are not case-sensitive, which is why I avoided uppercase characters. Do you think I should add a comment explaining that?

Comment (PotatoParser) [reply main.go:557]: Oh! :sweat_smile:  Case-insensitive definitely makes sense! No need for the comment as an explanation!

**Pull request merged**